scalableminds/webknossos
# Fix Android panning to move in plane

This PR fixes the in-plane movement triggered by panning on touch screen devices. The problem specifically occurred for android phones.

### URL of deployed dev instance (used for testing):
- https://fixandroidpanning.webknossos.xyz

### Steps to test:
- Open the dev instance on your mobile phone and simply view a dataset or open a tracing.
- Panning in one of the viewports should move the viewed plane of that viewport around.

### Issues:
- fixes #3780

------
